shell/components/SortableTable/selection.js

```diff
@@ -575,13 +575,38 @@ function _filter(map, disableAll = false) {
   return out;
 }
 
-function _execute(resources, action, args, opts = {}) {
+function _execute(resources, action, args, opts = {}, ctx) {
   args = args || [];
+
+  // New pattern for extensions - always call invoke
```

Comment:
This comment and functionality  isn't quite right. The example has `invoke` but also `multiple` so i guess should fall through to the next bit?

I tested selecting all rows in the App Repo table and clicking on an action, there's `console` output but no resources to act on (`invoke` prints out `[]` for `values`)

Reply:
I think that might have been a bug because of the lack of the `action` property. Can you confirm it again after my commit?